lunatrace/bsl/hasura/metadata/databases/lunatrace/tables/public_projects.yaml

```diff
@@ -27,13 +27,6 @@ array_relationships:
       table:
         name: ignored_vulnerabilities
         schema: public
-- name: github_repositories
```

Comment:
These were duped and the metadata was unable to be applied. I'm not sure exactly why -- probably a bad merge -- but this fixes it to remove the dupes. (Expand the file to verify. The "dupes" are 100% the same.)